hackathon_site/dashboard/frontend/src/api/api.ts

```diff
@@ -45,6 +45,11 @@ const makeConfig = () => ({
     withCredentials: true,
 });
 
+export const stripHostname = (url: string): string => {
+    const parsed = new URL(url);
```

Comment:
Had to actually switch to this since in production `SERVER_URL` will just be an empty string, but the API will come back with the full `https://example.com/api/...`.